testnet/stacks-node/src/mockamoto/tests.rs

```diff
@@ -72,20 +59,7 @@ fn observe_100_blocks() {
         info!("Block height observed: {stacks_block_height}");
 
         if stacks_block_height == 1 {
-            let tip = NakamotoChainState::get_canonical_block_header(chainstate.db(), &sortdb)
-                .unwrap()
-                .unwrap();
-            mempool
-                .submit_raw(
-                    &mut chainstate,
-                    &sortdb,
-                    &tip.consensus_hash,
-                    &tip.anchored_header.block_hash(),
-                    transfer_tx.clone(),
-                    &ExecutionCost::max_value(),
-                    &StacksEpochId::Epoch30,
-                )
-                .unwrap();
+            submit_tx(&http_origin, &transfer_tx);
```

Comment:
Can you make this a new test, like `mempool_rpc_submit()`, and leave the old test unchanged?

Reply:
Will do!